src/codeu/chat/server/JSON.java

```diff
@@ -0,0 +1,110 @@
+package codeu.chat.server;
+
+import java.io.File;
+import java.io.IOException;
+
+import codeu.chat.common.ConversationHeader;
+import codeu.chat.common.Message;
+import codeu.chat.common.User;
+
+import com.fasterxml.jackson.core.JsonGenerationException;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.util.DefaultPrettyPrinter;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectWriter;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+public final class JSON {
```

Comment:
Might think about adding JavaDoc to this class. Comments on functions, etc. I see that you've started that, but some of the comments are maybe outdated- referring to a user parameter that's not used?